glue/keyboard_shortcuts.py

```diff
@@ -0,0 +1,50 @@
+"""
+Jesse Averbukh
+October 12, 2017
+
+The file where keyboard shortcuts are created. In the future, many of these
+values will be populated using a GUI.
+"""
+
+from qtpy import QtCore, QtWidgets, QtGui, compat
+from glue.config import keybindings
+from glue.config import viewer_tool
+from glue.viewers.common.qt.data_viewer import DataViewer
+from glue.core import Data
+
+
+def check_duplicate_shortcut(key_shortcut):
+    """
+    Checks to make sure that a key_shortcut is not already used within the
+    glue application somewhere else.
+    This will become simpler with the implementation of a GUI
+    """
+    list_of_shortcuts = []
+    for k in viewer_tool.__iter__():
+        list_of_shortcuts.append(viewer_tool.members[k].shortcut)
+
+    if key_shortcut in list_of_shortcuts:
+        return True
+    return False
+
+
+@keybindings(QtCore.Qt.Key_Tab, ["2D Image", "1D Histogram", "2D Scatter", "Table Viewer"])
```

Comment:
Here you could list the classes directly for the viewers (you'll need to import them into this file)